home/import_helpers.py

```diff
@@ -165,16 +165,36 @@ def extract_errors(data: dict[str | int, Any] | list[str]) -> dict[str, str]:
     return error_message
 
 
-def fix_rows(rows: Generator[dict[str, str], None, None]) -> Iterator[dict[str, str]]:
+def fix_rows(
+    rows: Generator[dict[str, str], None, None] | Iterator[dict[str | Any, Any]]
```

Comment:
Since we're not feeding anything into the generator, I'm pretty sure we can skip that type and just use `Iterator[dict[str | Any, Any]`. Although by the time we get here, we should be guaranteed string keys at least.